<launch>

  <arg name="gui" default="false" />

  <arg name="input_cloud" value="/camera/depth/color/points" />

  <node name="attention_clipper"
        pkg="nodelet" type="nodelet"
        args="standalone jsk_pcl/AttentionClipper">
    <remap from="~input/points" to="$(arg input_cloud)" />
    <rosparam>
      initial_pos: [0, 0, 0]
      initial_rot: [0, 0, 0]
      dimension_x: 0.2
      dimension_y: 0.2
      dimension_z: 2
      frame_id: camera_depth_optical_frame
    </rosparam>
  </node>

  <node name="extract_indices"
        pkg="jsk_pcl_ros" type="extract_indices">
    <remap from="~input" to="$(arg input_cloud)" />
    <remap from="~indices" to="attention_clipper/output/point_indices" />
  </node>

</launch>

